### PR TITLE
Cred Man: add basic get tests

### PR DESCRIPTION
**Pull request by marcoscaceres**
Tests for https://github.com/w3c/webappsec-credential-management/pull/250

### CODE REVIEW

File: credential-management/credentialscontainer-get-basics.https.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<title>Credential Management API: create() basics.</title>
```
Comment (marcoscaceres) [inline credential-management/credentialscontainer-get-basics.https.html:2]:
```suggestion
<title>Credential Management API: get() basics.</title>
```

File: credential-management/credentialscontainer-get-basics.https.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      t,
+      "NotSupportedError",
+      navigator.credentials.get({ x: "y", y: "z" })
+    );
+
+    await promise_rejects_dom(
+      t,
+      "NotSupportedError",
+      navigator.credentials.get({ x: "y" })
+    );
```
Comment (nsatragno) [inline credential-management/credentialscontainer-get-basics.https.html:33]: isn't this the same as L17?

**Review by nsatragno — APPROVED** [review]
Approved with a comment.

File: credential-management/credentialscontainer-get-basics.https.html (hunk 2 above)
Comment (marcoscaceres) [reply credential-management/credentialscontainer-get-basics.https.html:33]: Can't ever over test! 🤭 ... will remove

Comment (marcoscaceres) [inline credential-management/credentialscontainer-get-basics.https.html:33]:
```suggestion
```

File: credential-management/credentialscontainer-get-basics.https.html [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+    await promise_rejects_dom(
+      t,
+      "NotSupportedError",
+      navigator.credentials.get({ x: "y", y: "z" })
+    );
+
+
```
Comment (marcoscaceres) [inline credential-management/credentialscontainer-get-basics.https.html:29]:
```suggestion

```

File: credential-management/credentialscontainer-get-basics.https.html [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+      "NotSupportedError",
+      navigator.credentials.get({ x: "y" })
+    );
+
+    await promise_rejects_dom(
+      t,
+      "NotSupportedError",
+      navigator.credentials.get({ x: "y", y: "z" })
+    );
+    await promise_rejects_dom(
```
Comment (marcoscaceres) [inline credential-management/credentialscontainer-get-basics.https.html:28]:
```suggestion
    
    await promise_rejects_dom(
```

**Review by nsatragno — APPROVED** [review]
Approved, thank you!

Can't land yet due to the trailing whitespace (check the linter), but you should be good after that.

**Pull request merged**